addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -9078,7 +9072,7 @@ public void db_viewCustomTitles(int client, char[] szSteamID)
 	Handle pack = CreateDataPack();
 	WritePackCell(pack, client);
 	WritePackString(pack, szSteamID);
-	Format(szQuery, 728, "SELECT `title`, `namecolour`, `textcolour`, `inuse`, `vip`, `zoner`, `joinmsg` FROM `ck_vipadmins` WHERE `steamid` = '%s'", szSteamID);
+	Format(szQuery, 728, "SELECT `title`, `namecolour`, `textcolour`, `inuse`, `vip`, `zoner`, `joinmsg` FROM `ck_vipadmins` WHERE `steamid` = '%s';", szSteamID);
```

Comment:
are you sure about this semicolon? 😉

Reply:
I do believe it needs to be there, yes. I have not explicitly tested this exact functionality, but all  the format() function calls within sql.sp either specify an existing query initialized in queries.sp or they provide an inline query inside double quotes, as is the case here. As such it should need the terminating semicolon.